.changeset/shaggy-flowers-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': patch
+---
+
+Removed the border-radius from the tertiary `Button`.
```

Comment:
According to specs this should still have a border radius. What do you mean about clipping? Maybe I can help find a workaround?

<img width="744" alt="Figma spec for Circuit UI buttons and their states, highlighting the rounded border-radius for a tertiary button" src="https://user-images.githubusercontent.com/35560568/185732524-34d57825-6b75-4848-9ff3-7a5a6bc13960.png">

Reply:
Ah right, I forgot about the focus outline! I was wondering why we hadn't removed it before 🤦‍♂️

On the NotificationInline action button, the border-radius clips the button label when removing the bottom padding. I can work around that by keeping the padding and using a negative margin instead.

Reply:
Done in 81fdd32.